Ref/TypeDemo/TypeDemo.fpp

```diff
@@ -31,6 +31,8 @@ module Ref {
         separateChoice: Choice
         @ A pair of choices
         choicePair: ChoicePair
+        @ An array of choices defined as member array
+        choiceAsMemberArray: [2] U8
```

Comment:
I apply your suggestion. Please note that before merging this, the [related pull request in fprime-gds](https://github.com/fprime-community/fprime-gds/pull/161) should be merged otherwise `fprime-gds` will not work as expected with the new member array.